connector_acp_ftp/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# Copyright (C) 2019 Open Source Integrators
+# Copyright (C) 2019 Serpent Consulting Services Pvt. Ltd.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+{
+    "name": "FTP ACP Connector",
+    "version": "12.0.1.0.0",
+    "license": "AGPL-3",
+    "category": "Localization",
+    "summary": "Support FTP protocol as a connector backend.",
+    "author": "Open Source Integrators, Odoo Community Association (OCA)",
```

Comment:
```suggestion
    "author": "Open Source Integrators, "
    "Serpent Consulting Services, "
    "Odoo Community Association (OCA)",
```

Reply:
Black decided this way, so I prefer to stick to black's output.

Reply:
@dreispt :+1: but can you add Serpent?